Fix static scan issue: dead code and unused value

jiaxuan-guo opened the pull request:
OAM-124684

GangSecurity commented on drivers/dw3/dw3.c:
```diff
@@ -647,6 +647,9 @@ UsbdSetupHdlr (
 	                        Status = UsbdSetAddress ((UINT8)CtrlRequest->Value);
+							if (EFI_ERROR (Status)) {
+	              				DEBUG ((DEBUG_INFO, "UsbdSetAddress: EFI_DEVICE_ERROR\n"));
+	        				}
```
no return ?

jiaxuan-guo replied:
Prefer to keep the previous code flow since this might just be a warning.

GangSecurity commented on drivers/dw3/dw3.c:
```diff
@@ -590,6 +590,9 @@ UsbdSetupHdlr (
 	                                        &(mCtrlIoReq.IoInfo.Length)
 	                                        );
+									if (EFI_ERROR (Status)) {
+										DEBUG ((DEBUG_INFO, "UsbdGetStringDesc() - EFI_DEVICE_ERROR"));
```
return?

jiaxuan-guo replied:
same

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

jiaxuan-guo committed:
Fix logically dead code

1. cmd couldn't be NULL
2. redundant check for EfiResetShutdown

Tracked-On: OAM-124684
Signed-off-by: Guo, Jade <jade.guo@intel.com>

jiaxuan-guo committed:
Fix unused value

The return values of some functions are ignored. Add DEBUG log while continue
the code flow without quit early.
Include: UsbdSetAddress UsbdGetStringDesc UsbdGetConfigDesc UsbdGetConfig UsbdSetConfig
UsbdGetStatus VirtioRpmbInitialize

Tracked-On: OAM-124684
Signed-off-by: Guo, Jade <jade.guo@intel.com>

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon_umr0_master-merge/118
